docs/notebooks/plot_cap.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<p>I would add just one or two sentences about the dataset. Something along the lines of "this is the famous mtcars dataset which contains..." bla bla bla. </p><p> </p><p>Or you can simply copy the description</p><p> </p><p>The data was extracted from the 1974 Motor Trend US magazine, and comprises fuel consumption and 10</p><p>aspects of automobile design and performance for 32 automobiles (1973--74 models).&nbsp;</p><p>The following is a description of the variables:</p><p> </p><p>* mpg: Miles/(US) gallon</p><p>* cyl: Number of cylinders</p><p>* disp: Displacement (cu.in.)</p><p>* hp: Gross horsepower</p><p>* drat: Rear axle ratio</p><p>* wt: Weight (1000 lbs)</p><p>* qsec: 1/4 mile time</p><p>* vs: Engine (0 = V-shaped, 1 = straight)</p><p>* am: Transmission (0 = automatic, 1 = manual)</p><p>* gear: Number of forward gear</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>